src/vulkan-renderer/renderer.cpp

```diff
@@ -158,18 +158,20 @@ void VulkanRenderer::calculate_memory_budget() {
 
 VulkanRenderer::~VulkanRenderer() {
     spdlog::debug("Shutting down vulkan renderer");
-    // TODO: Add wrapper::Device::wait_idle()
-    vkDeviceWaitIdle(m_device->device());
```

Comment:
Same here :sweat_smile: (just looking for the early return in this case)

Reply:
Done.